core/templates/pages/community-dashboard-page/community-dashboard-page.component.spec.ts

```diff
@@ -76,11 +76,6 @@ describe('Community dashboard page', function() {
       $rootScope.$apply();
     });
 
-    it('should get username', function() {
```

Comment:
Why was this deleted?

Reply:
This spec's assertion is being evaluated in another spec.